### PR TITLE
New vscode tests

### PR DESCRIPTION
**Pull request by tisnik**
Implements checks for basic VSCode Analytics plugin behavior.

### CODE REVIEW

File: vscode-visual-tests/features/analytics_extension_check_on_cli.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: Visual Studio Code + the Analytics plugin basic functionality
+
+    Scenario: Check that the Analytics plugin can be found in the Marketplace
+        Given The PyAutoGUI library is initialized
+          And The screen resolution is at least 1024x768 pixels
```
Comment (miteshvp) [inline vscode-visual-tests/features/analytics_extension_check_on_cli.feature]: Do we need this minimum resolution? And it is at many places. Is there any such limit by VsCode?

File: vscode-visual-tests/features/analytics_extension_install_hla.feature [hunk 2]
```diff
@@ -0,0 +1,92 @@
+Feature: Installation and uninstallation of Analytics pluging for the Visual Code Studio
```
Comment (miteshvp) [inline vscode-visual-tests/features/analytics_extension_install_hla.feature]: Typo - pluging -> plugin

File: vscode-visual-tests/features/analytics_extension_check_on_cli.feature (hunk 1 above)
Comment (tisnik) [reply vscode-visual-tests/features/analytics_extension_check_on_cli.feature]: Yes, it is to ensure that the layout will stay the same. On smaller screen (but who have less than 1024x768 nowadays?) the layout was not consistent.

File: vscode-visual-tests/features/analytics_extension_install_hla.feature (hunk 2 above)
Comment (tisnik) [reply vscode-visual-tests/features/analytics_extension_install_hla.feature]: nice catch!

**Pull request merged**